skyvern/library/skyvern.py

```
@@ -186,19 +187,31 @@ def local(
+            do_use_in_memory_db = use_in_memory_db
+        else:
+            do_use_in_memory_db = os.environ.get("SKYVERN_USE_IN_MEMORY_DB", "").lower() == "true"
+
+        if not do_use_in_memory_db and not os.path.exists(".env"):
+            raise ValueError("Please run `skyvern quickstart` to set up your local Skyvern environment.")
+
+        if not llm_config and not os.path.exists(".env"):
+            raise ValueError(
+                "LLM configuration is required. Either provide llm_config parameter or "
+                "run `skyvern quickstart` to configure LLM settings in .env file."
+            )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)